docs/source/backend/architecture.rst

```diff
@@ -0,0 +1,18 @@
+Architecture
+============
+
+The os2phonebook backend consists of two components:
+
+* A Python program (located in the ``import`` directory) that imports
+  all the necessary data from MO and stores it in the file system, and
+* An SOLR server in which the data are indexed, and from which the
```

Comment:
I believe “SOLR” is pronounced “solar” 🙂
```suggestion
* A SOLR server in which the data are indexed, and from which the
```

Reply:
:+1: